test/vc-data-model-1.0/input/example-015-zkp-vp.jsonld

```diff
@@ -0,0 +1,33 @@
+{
+  "@context": [
+    "https://www.w3.org/2018/credentials/v1",
+    "https://www.w3.org/2018/credentials/examples/v1"
+  ],
+  "type": "VerifiablePresentation",
+  "verifiableCredential": {
+    "@context": [
+      "https://www.w3.org/2018/credentials/v1",
+      "https://www.w3.org/2018/credentials/examples/v1"
+    ],
+    "type": ["VerifiableCredential", "UniversityDegreeCredential"],
+    "credentialSchema": {
+      "id": "did:example:cdf:35LB7w9ueWbagPL94T9bMLtyXDj9pX5o",
+      "type": "did:example:schema:22KpkXgecryx9k7N6XN1QoN3gXwBkSU8SfyyYQG"
+    },
+    "issuer": "did:example:Wz4eUg7SetGfaUVCn8U9d62oDYrUJLuUtcy619",
+    "issuanceDate": "2010-01-01T19:23:24Z",
+    "credentialSubject": {
+      "degreeType": "BachelorDegree",
+      "degreeSchool": "College of Engineering"
+    },
+    "proof": {
+      "type": "ex:AnonCredDerivedCredentialv1",
+      "primaryProof": "cg7wLNSi48K5qNyAVMwdYqVHSMv1Ur8i...Fg2ZvWF6zGvcSAsym2sgSk737",
+      "nonRevocationProof": "mu6fg24MfJPU1HvSXsf3ybzKARib4WxG...RSce53M6UwQCxYshCuS3d2h"
```

Comment:
Any term (e.g. `primaryProof`, `nonRevocationProof`) should be defined in the core context or the example context. It seems like these are just example terms for testing, so we should add them to the example context. We can do the same with `AnonCredDerivedCredentialv1` in order to remove the `ex:` prefix if we want. The same goes for `degreeType` and `degreeSchool` used above. The alternative to adding new terms to the example context is to use whatever is already in there, if it applies to the use case.

cc: @dmitrizagidulin